src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java

```diff
@@ -292,11 +294,21 @@ default float getXpRepairRatio()
     /**
      * Called to tick armor in the armor slot. Override to do something
      */
+    @Deprecated(forRemoval = true, since = "1.20.1") // Use onInventoryTick
     default void onArmorTick(Level level, Player player)
     {
         self().getItem().onArmorTick(self(), level, player);
     }
 
+    /**
+     * @see {@link IForgeItem#inventoryTick(ItemStack, Level, Entity, NonNullList, int, ResourceLocation, int)}
```

Comment:
Personally I prefer being able to see the javadocs directly rather than just proxying them so once we figure out the full javadocs of the `IForgeItem` method we maybe should just copy bits of them here? Unless we decide that going forward it makes more sense for Neo to just have see/reference from stack/state to the item/block